docs/admin/integration/casa.md

```diff
@@ -1,22 +1,40 @@
-**Casa** is a self service web portal that allows end users to view, enroll, and remove MFA credentials, including hardware tokens, software tokens, commercial services (like Duo), social login, biometric, and mobile.  Casa is also extensible. As new authentication technologies arise, you can download plugins to leverage them in your organization through Casa
+# Casa - Self-service MFA portal
+## Overview
 
-***
+**Casa** is a self-service web portal that allows end users to view, enroll, and remove MFA credentials, including hardware tokens, software tokens, commercial services (like Duo), social login, biometric, and mobile.  Casa is also extensible. As new authentication technologies arise, you can download plugins to leverage them in your organization through Casa
+
+<p align="center">
+<img src="../../assets/image-casa-module-diagram.png">
+</p>
+
+## Prerequisites
+- Installed Janssen Server  
 
 ## Installing Casa
+Casa gets installed as an add-on to Janssen Server and integrates with `jans-auth-server` module as an OpenID connect Relying Party.
```

Comment:
Im a VM currently no. Casa lives on the same VM I.e it requires the Janssen server to be installed beforehand or else it will install it. What we are referring to is that it doesn't live in the Janssen server but rather in the VM that holds the Janssen server . This makes it look like Casa might be a part of Janssen and that's not true . You need to add a layer block behind Janssen server and call that deployment perhaps add flex block in that and casa in Flex block that connects to the Janssen server block .

Reply:
Updated the diagram to reflect the same.